airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```diff
@@ -7,11 +7,11 @@ readme = "README.md"
 packages = [{include = "orchestrator"}]
 
 [tool.poetry.dependencies]
-python = "^3.9" # This is set to 3.9 as currently there is an issue when deploying via dagster-cloud where a dependency does not have a prebuild wheel file for 3.10
-dagit = "^1.4.1"
-dagster = "^1.4.1"
+python = "^3.9, <3.13" # This is set to 3.9 as currently there is an issue when deploying via dagster-cloud where a dependency does not have a prebuild wheel file for 3.10
```

Comment:
JW, this note says its "set to 3.9" but this pinning actually means we support 3.9.x-3.12.x. Do you know if 3.10 is actually a problem, and if so, if we should actually be pinning it to 3.9 specifically?